mergebot_backend/mergebot_poller.py

```diff
@@ -125,6 +125,10 @@ def poll(self):
                     self.publisher.publish_poller_status(status='SHUTDOWN')
                     return
             self.publisher.publish_poller_heartbeat()
```

Comment:
Update comment on get_authorized_users to say it returns a dict from committer user id to github user id

Optionally, being able to differentiate between empty set of authorized users and an error occurred while looking up users would be good.